libgadget/sfr_eff.c

```diff
@@ -529,26 +558,30 @@ starformation(int i, double *localsfr, double * sum_sm)
 
 double get_starformation_rate(int i) {
     /* returns SFR in internal units */
-    return get_starformation_rate_full(i, 0, NULL, NULL, NULL);
+    return get_starformation_rate_full(i, 0, NULL, NULL, NULL, NULL);
 }
 
-static double get_starformation_rate_full(int i, double dtime, MyFloat * ne_new, double * trelax, double * egyeff) {
+static double get_starformation_rate_full(int i, double dtime, MyFloat * ne_new, double * trelax, double * egyhot_i, double * coolfrac) {
```

Comment:
egyhot_out, or rename the local variable to avoid a name conflict?

Also, coolfrac really is cloud_fraction, if you look at line 610.